client/src/app/+admin/follows/followers-list/followers-list.component.html

```diff
@@ -25,7 +25,7 @@
   <ng-template pTemplate="body" let-follow>
     <tr>
       <td>{{ follow.id }}</td>
-      <td>{{ follow.follower.name + '@' + follow.follower.host }}</td>
+      <td><a href="{{follow.follower.url}}">{{ follow.follower.name + '@' + follow.follower.host }}</a></td>
```

Comment:
`[href]=follow.follower.url`

And use `target="_blank" rel="noopener noreferrer"` to open the link on another tab please

Reply:
Thank you for the review. I'm on it.